sanic/models/asgi.py

```diff
@@ -27,9 +27,9 @@ def __init__(self, transport: "MockTransport", loop):
             self._not_paused.set()
             self._complete = asyncio.Event()
         else:
-            self._not_paused = asyncio.Event(loop=loop)
+            self._not_paused = asyncio.Event(loop=loop) # type: ignore
             self._not_paused.set()
-            self._complete = asyncio.Event(loop=loop)
+            self._complete = asyncio.Event(loop=loop) # type: ignore
```

Comment:
What is the type error? I would rather fix the typing than ignore it.

Reply:
`Python 3.11.3`: error: Unexpected keyword argument "loop" for "Event"